STEPS_API.md

```diff
@@ -0,0 +1,255 @@
+- [configure_instance](#configure_instance)
+- [restart_instance](#restart_instance)
+- [wait_instance_started](#wait_instance_started)
+- [connect_to_membership](#connect_to_membership)
+- [set_control_instance](#set_control_instance)
+- [edit_topology](#edit_topology)
+- [cleanup_expelled](#cleanup_expelled)
+- [configure_auth](#configure_auth)
+- [configure_app_config](#configure_app_config)
+- [bootstrap_vshard](#bootstrap_vshard)
+- [configure_failover](#configure_failover)
+
+### deliver_package
+
+Delivery of the application package to physical machines.
+
+Input facts (set by role):
+- `single_instances_for_each_machine` - list of Ansible instance names (one for each physical machine).
+
+Input facts (set by config):
+- `cartridge_package_path` - path to file of package to delivery.
+
+Output facts:
+- `delivered_package_path` - path to file of delivered package.
+
+### update_package
+
+Install the delivered package on physical machines.
+
+Input facts (set by role):
+- `single_instances_for_each_machine` - list of Ansible instance names (one for each physical machine);
+- `delivered_package_path` - path to file of delivered package.
+
+Input facts (set by config):
+- `cartridge_app_name` - application name;
+- `cartridge_enable_tarantool_repo` - indicates if the Tarantool repository should be enabled.
+
+Output facts:
+- `package_info` - information about delivered package. It's a dictionary with fields:
+  - `name` - application name;
+  - `tnt_version` - version of Tarantool required by application;
+  - `package_type` - type of package (`rpm` or `deb`).
+
+### configure_instance
+
+Configure instance in runtime and change instance config.
+
+Input facts (set by role):
+- `instance_info` - information for a current instance. It's a dictionary with fields:
+  - `console_sock` - path to control socket of instance;
+  - `app_conf_file` - path to file with application config;
+  - `conf_section` - name of section in instance config file;
+  - `conf_file` - path to file with instance config;
+  - `instance_code_dir` - path to directory with distribution files.
+
+Input facts (set by config):
+- `config` - instance configuration;
+- `restarted` - if instance should be restarted or not (user forced decision);
+- `expelled` - indicates if instance must be expelled from topology;
+- `stateboard` - indicates that the instance is a stateboard;
+- `cartridge_app_name` - application name;
+- `cartridge_cluster_cookie` - cluster cookie for all cluster instances;
+- `cartridge_defaults` - default configuration parameters values for instances.
+
+Output facts:
+- `needs_restart` - if instance should be restarted to apply code or configuration changes.
+
+### restart_instance
+
+Restart instance if it should be restarted.
+
+Input facts (set by role):
+- `needs_restart` - if instance should be restarted to apply code or configuration changes;
+- `instance_info` - information for a current instance. It's a dictionary with fields:
```

Comment:
I think that we shouldn't describe `instance_info` fields here, just add a link to it.